napari/components/tests/test_multichannel.py

```diff
@@ -1,15 +1,15 @@
 import numpy as np
 from napari.components import ViewerModel
 
-base_colormaps = ['cyan', 'yellow', 'magenta', 'red', 'green', 'blue']
+base_colormaps = ['red', 'green', 'blue', 'cyan', 'yellow', 'magenta']
```

Comment:
@sofroniewn there's been a lot of fuss about RGB not being colorblind friendly compared to CYM. On the other hand, I understand that it's more surprising to default to that. So, I don't feel strongly about undoing this change but I still lean towards CYM. What do you think?

Reply:
I should have left a note explaining this, the reason for the change is that with this ordering if someone opens an rgb image (say jpeg or png) and adds it with `axis=2` they will get an image that looks the same as the original but with the layers separated.

For example now
```python
napari.view_image(data.astronaut(), channel_axis=2)
```
will yield:
<img width="1200" alt="multichannel_rgb" src="https://user-images.githubusercontent.com/6531703/67650519-76ca6980-f8fa-11e9-8c66-423bbb79288b.png">

If we change to CYM we get:
<img width="1200" alt="multichannel_cym" src="https://user-images.githubusercontent.com/6531703/67650555-9d88a000-f8fa-11e9-8e9f-6af6d7764697.png">

which I think will be quite confusing to people.

I appreciate that in all other cases (say fluorescent microscopy data) CYM or GMBCRY as suggested by @tlambert03 is a better option.

On option could be to try and do some special casing say is axis is len(3) use RGB, otherwise go with GMBCRY, but that also might be confusing.

Another point of view is to say that we primarily expect people to use the channel_axis keyword arg with data that is not rgb to begin with (or where those colors don't have intrinsic meaning)

What are your thoughts @royerloic @jni @tlambert03

Reply:
After reading more of the comments below again, I think I'm leaning towards `RGB` if length 3 and `GMBCRY` otherwise

Reply:
@sofroniewn well, we haven't removed `rgb=` here yet, have we? So I suggest that users with rgb images should use that keyword argument. Then if we have `rgb=True, channel_axis=2` we use multichannel rgb colormap, and if `channel_axis=None` we use the rgb display.

napari is primarily for scientific images, so I would very much prefer to cater to microscopy display than to natural image display, as a general rule, especially since we already provide a keyword argument for RGB.

Reply:
I’d rather keep the rgb parameter as is so that someone could use channel_axis to expand a list of rgb images if they wanted, though maybe then it should be called something like layer_axis, or expand_axis

As to scientific use case first, I agree, I can go with the GMBCRY order as default for now, we can always give people the option to change that default somewhere else in the future. Thoughts @jni?

Reply:
I think it's quite natural that if someone passes `rgb=True` and `channel_axis=[anything other than None]`, they mean to expand the RGB channels into layers. Currently what would happen? Would it break?

Whether we want that to be the recommended way of doing things is another matter. We can always add a docs example with `colormap=['red', 'green', 'blue']`, and have that be the "canonical" way of doing it, until a global config is defined.

Reply:
for `rgb=True`, if you have data that is `[100, 100, 5, 3]` and you split along `channel_axis=2` you'll get 5 rgb images that are 100x100. I think this is the expected behavior, and I've added a test to confirm it works

I am going to go with `GM` if len < 3, and `CYMRGB` otherwise. we can expose more default settings another time